example-secret.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
@betaredex Same comment here as in pfioh.  I think the readme needs updates based on these changes.

Reply:
Will do.